logzio-sender-test/src/main/java/io/logz/test/MockLogzioBulkListener.java

```diff
@@ -86,8 +87,10 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
                         logger.debug("got log: {} ", line);
                     });
                     logger.debug("Total number of logRequests {} ({})", logRequests.size(), logRequests);
-
-                    // Tell Jetty we are ok, and it should return 200
+                } catch (IllegalArgumentException e) {
```

Comment:
Shouldn't this be a Jackson exception here?

Reply:
Why? I'm catching exceptions from `LogRequest` function